go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/ONSdigital/dp-frontend-homepage-controller
+
+go 1.12
```

Comment:
make it `go 1.13`, we're standardising on thatversion and it matches the version of go used to build the service.

Reply:
👍